site/src/components/Form/Form.tsx

```diff
@@ -81,59 +82,65 @@ interface FormSectionProps {
   deprecated?: boolean;
 }
 
-export const FormSection: FC<FormSectionProps> = ({
-  children,
-  title,
-  description,
-  classes = {},
-  alpha = false,
-  deprecated = false,
-}) => {
-  const { direction } = useContext(FormContext);
-  const theme = useTheme();
-
-  return (
-    <div
-      css={{
-        display: "flex",
-        alignItems: "flex-start",
-        flexDirection: direction === "horizontal" ? "row" : "column",
-        gap: direction === "horizontal" ? 120 : 24,
-
-        [theme.breakpoints.down("md")]: {
-          flexDirection: "column",
-          gap: 16,
-        },
-      }}
-      className={classes.root}
-    >
-      <div
+export const FormSection = forwardRef<HTMLDivElement, FormSectionProps>(
+  (
+    {
+      children,
+      title,
+      description,
+      classes = {},
+      alpha = false,
+      deprecated = false,
+    },
+    ref,
+  ) => {
+    const { direction } = useContext(FormContext);
+    const theme = useTheme();
+
+    return (
+      <section
+        ref={ref}
         css={{
-          width: "100%",
-          maxWidth: direction === "horizontal" ? 312 : undefined,
-          flexShrink: 0,
-          position: direction === "horizontal" ? "sticky" : undefined,
-          top: 24,
+          display: "flex",
+          alignItems: "flex-start",
+          flexDirection: direction === "horizontal" ? "row" : "column",
+          gap: direction === "horizontal" ? 120 : 24,
```

Comment:
emotion prefers when stuff like this is done as a separate conditionally applied class; something like:

```ts
css={[
  {
    display: "flex",
    alignItems: "flex-start",
    flexDirection: "column",
    gap: 24,

    [theme.breakpoints.down("md")]: {
      flexDirection: "column",
      gap: 16,
    },
  },
  direction === "horizontal" && {
    flexDirection: "row",
    gap: 120,
  }
]}
```